src/20_Components/amp-bind.html

```diff
@@ -259,6 +259,17 @@
     </amp-img>
   </section>
 
+    <!-- ## Refresh state -->
+
+    <!-- `amp-state` supports the `refresh` action. This can be useful in many cases, for example the initial URL loading fails
```

Comment:
`This can be useful in many cases, for example the initial URL loading fails and you want the user to load the state again.`

Add: "Another common scenario is live content that needs to be updated, e.g. a sports live score."